pkg/devspace/config/versions/latest/schema.go

```diff
@@ -817,6 +819,38 @@ type RollingUpdateConfig struct {
 	Partition      *int   `yaml:"partition,omitempty" json:"partition,omitempty"`
 }
 
+// TankaConfig defines the specific tanka options used during deployment.
+type TankaConfig struct {
+	// Path is the (relative) path of the tanka environment, usually identified by jsonnetfile.json.
+	Path string `yaml:"path,omitempty" json:"path,omitempty"`
+	// RunJsonnetBundlerInstall indicates if the `jb install` command shall be run
+	RunJsonnetBundlerInstall bool `yaml:"runJsonnetBundlerInstall,omitempty" json:"runJsonnetBundlerInstall,omitempty"`
```

Comment:
Shouldn't this be set to true by default?

Reply:
See https://github.com/loft-sh/devspace/pull/2344#discussion_r1054291404